app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
```

Comment:
@Meltrust 
1) Added the `:comments` and `:likes` associations.
2) `:user` symbol is just a naming convention for Rails to use in routes or views, there is no such information on the diagram about this but i think it wont be a bad idea to make it `author` since it will make it easier to read it.